.gitignore

```diff
@@ -596,4 +596,5 @@ healthchecksdb
 # Backup folder for Package Reference Convert tool in Visual Studio 2017
 MigrationBackup/
 
-# End of https://www.gitignore.io/api/git,dart,flutter,intellij,webstorm,visualstudio
\ No newline at end of file
+# End of https://www.gitignore.io/api/git,dart,flutter,intellij,webstorm,visualstudio
+*.env
```

Comment:
Is this deliberate? 👀

Reply:
the order is wrong :D the addition is deliberate, it should be before the "End" comment.